src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ComboBoxTests/UnoSamples_Tests.ComboBoxTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.ComboBoxTests
+{
+	[TestFixture]
+	public partial class ComboBoxTests_Tests : SampleControlUITestBase
+	{
+		[Test]
+		public void ComboBoxTests_DefaultValue()
+		{
+			Run("SamplesApp.Wasm.Windows_UI_Xaml_Controls.ComboBox.ComboBox_ItemsSource");
+
+			_app.WaitForElement(_app.Marked("theComboBox"));
+			var theComboBox = _app.Marked("theComboBox");
+
+			// Assert initial state
+			var myDevice = _app.Device.GetType();
+			if (_app.Device.GetType().Name.Contains("Android") || _app.Device.GetType().Name.Contains("iOS"))
```

Comment:
It seems strange that you need this, is this because Uno behaves differently on WASM, or because the `GetDependencyPropertyValue()` method behaves differently? Can you add a comment explaining please.